bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -52,15 +52,29 @@ def select_parameters(self) -> None:
         Opens a dialog to select the parameters.json configuration to use.
         """
 
-        response = self.get_filename_dialog(message='Select parameters file', file_type='JSON files (*.json)')
+        response = self.get_filename_dialog(message='Select parameters file',
+                                            file_type='JSON files (*.json)')
         if response:
             self.set_parameter_location(response)
+            # If outdated, prompt to merge with the current defaults
+            default_parameters = load_json_parameters(DEFAULT_PARAMETERS_PATH,
```

Comment:
Unrelated to the PR... I was thinking of making value cast true by default. It seems that is the most common case, and I don't even think we call it anywhere without casting. What do you think?